lib/Core/ExecutorUtil.cpp

```diff
@@ -33,6 +33,75 @@ using namespace llvm;
 
 namespace klee {
 
+  ref<klee::ConstantExpr> Executor::evalConstant(const Constant *c) {
```

Comment:
This can't be part of `Executor` because you've removed it from the class declaration of `Executor`.